frontend/src/metabase/selectors/app.ts

```diff
@@ -18,14 +18,17 @@ export interface RouterProps {
 }
 
 const HOMEPAGE_PATH = /^\/$/;
+const AUTH_PATH = /^\/auth/;
 const PATHS_WITHOUT_NAVBAR = [
+  AUTH_PATH,
   /\/model\/.*\/query/,
   /\/model\/.*\/metadata/,
   /\/model\/query/,
   /\/model\/metadata/,
 ];
 const EMBEDDED_PATHS_WITH_NAVBAR = [
   HOMEPAGE_PATH,
+  AUTH_PATH,
```

Comment:
I think we just bait ourselves. We have one variable name `*_WITHOUT_NAVBAR` and another with `*_WITH_NAVBAR` notice **with** vs **without**?

So you should never put `AUTH_PATH` at line 31, although the embedding still behaves the same from my testing. This is only for code correctness.

Reply:
Great catch again! Fixed